config/maps.txt

```diff
@@ -27,7 +27,7 @@ map boxstation
 endmap
 
 map metastation
-	default
+	minplayers 40
```

Comment:
Change this back

Reply:
yea i forgot how that stood since i changed it out or testing wops

Reply:
Figured as much